### PR TITLE
Add sentry guide + sentry performance monitoring setup

### PR DESCRIPTION
**Pull request by theskumar**
- Add a guide on how to enable and setup sentry
- Upgrades sentry-sdk to 1.16.0
- Add optional performance monitoring
- Add Browser Javascript Tracking & Tracing

### CODE REVIEW

File: hypha/templates/base.html [hunk 1]
```diff
@@ -211,5 +211,25 @@ <h4 class="heading heading--no-margin">@{{ settings.utils.SocialMediaSettings.tw
...
+                    environment: "{{ SENTRY_ENVIRONMENT }}",
+                    debug: {% if SENTRY_DEBUG %}true{% else %}false{% endif %},
+                    integrations: [new BrowserTracing()],
+                    denyUrls: {{ SENTRY_DENY_URLS|safe }},
+                    tracesSampleRate: {{ SENTRY_TRACES_SAMPLE_RATE }}
+                });
+             });
+        </script>
+        {% endif %}
+        {% endblock sentry_sdk %}
```
Comment (frjo) [inline hypha/templates/base.html:233]: Is this not also needed in `base-apply.html`?

Should you have the block name in the end tag?

Comment (theskumar) [reply hypha/templates/base.html:233]: 1. yes. I always forget to add it to the base-apply.html
2. block name, in the end, is allowed, and make things clear which block is getting terminated.

Comment (theskumar) [reply hypha/templates/base.html:233]: Added.

**Comment by frjo** [comment]
@theskumar To me it looks like this could be merged in directly, agree?

**Comment by theskumar** [comment]
I was just testing the user feedback widget. Will mark as ready after

**Comment by theskumar** [comment]
The user feedback widget will be considered later. I need to look deeper into its integration with the new sdk. Good to merge and test now.

**Comment by frjo** [comment]
Personally I find all the chat/feedback widgets that are inserted on so many pages today rather annoying. Having a feedback page that is clearly linked in e.g. the footer is better in my opinion.

**Pull request merged**